Source/Core/DolphinQt/Host.h

```diff
@@ -27,6 +27,11 @@ class Host final : public QObject
   bool GetRenderFocus();
   bool GetRenderFullFocus();
   bool GetRenderFullscreen();
+  std::pair<double, double> GetRenderWindowDimensions();
+
+  //@return std::pair<double, double>{x,y} of top left corner of window
```

Comment:
This isn't clear what it means. The `x` and `y` values are relative to the top-left, yes, but in what scale? In pixels? Relative to the screen size, with `1/1` being the bottom-right?